apps/server/src/shared/repo/task/task.repo.ts

```diff
@@ -119,14 +123,24 @@ export class TaskRepo {
 			scope.byFinished(filters.finished.userId, filters.finished.value);
 		}
 
-		if (filters?.draft !== undefined) {
-			scope.byDraft(filters.draft);
+		if (parentIds.creatorId) {
```

Comment:
should the different cases for the existence of creatorId be grouped together instead of several if cases?

Reply:
the intend was to differentiate between the parents and the filters. this here is more about the filters.